.gitignore

```diff
@@ -0,0 +1,11 @@
+/target/
+/.settings/
+/.vscode/
+/.idea/
+/*.iml
+/.classpath
+/.factorypath
+/.flattened-pom.xml
+fuzzer/**
+output.log
```

Comment:
remove fuzzer and output.log, not used here

Reply:
Removed.